smartsim/_core/generation/generator.py

```diff
@@ -108,7 +108,16 @@ def _log_file(log_path: pathlib.Path) -> pathlib.Path:
         """
         return pathlib.Path(log_path) / "smartsim_params.txt"
 
-    def generate_job(self, job: Job, job_index: int) -> pathlib.Path:
+    def _output_files(
+        self, log_path: pathlib.Path, job_name: str
+    ) -> t.Tuple[pathlib.Path, pathlib.Path]:
+        out_file_path = log_path / (job_name + ".out")
+        err_file_path = log_path / (job_name + ".err")
```

Comment:
nit: I prefer string formatting over manual concatenation (in theory it's more performant, in practice I just think it's easier to read, lol)
```suggestion
        out_file_path = log_path / f"{job_name}.out")
        err_file_path = log_path / f"{job_name}.err")
```

Reply:
I agree I will adopt